app/containers/MessageBox/index.js

```diff
@@ -218,13 +241,32 @@ class MessageBox extends Component {
 		const result = msg.substr(0, cursor).replace(regexp, '');
 		const mentionName = trackingType === MENTIONS_TRACKING_TYPE_EMOJIS
 			? `${ item.name || item }:`
-			: (item.username || item.name);
+			: (item.username || item.name || item.command);
 		const text = `${ result }${ mentionName } ${ msg.slice(cursor) }`;
+		if ((trackingType === MENTIONS_TRACKING_TYPE_COMMANDS) && item.providesPreview) {
+			this.showCommandPreview = true;
+		}
 		this.setInput(text);
 		this.focus();
 		requestAnimationFrame(() => this.stopTrackingMention());
 	}
 
+	onPressCommandPreview = (item) => {
+		const { rid } = this.props;
+		const { text } = this;
+		const command = text.substr(0, text.indexOf(' ')).slice(1);
```

Comment:
What does this do?

Reply:
it takes the name of the command out item is the whole string now `text.substr(0, text.indexOf(' ')) ` gives /giphy `slice(1)` gives giphy

Reply:
What? Please rephrase your sentence.

Reply:
item is the whole text in the text input I need command as a param so I am getting that from this
ex-> item is /giphy no man can see
command = giphy